api/app/Http/Controllers/Api/EventTypeController.php

```diff
@@ -27,9 +29,11 @@ public function __construct(EventTypePresenter $eventTypePresenter)
         $this->eventTypePresenter = $eventTypePresenter;
     }
 
-    public function index(GetEventTypeCollectionAction $action)
+    public function index(Request $request, GetEventTypeCollectionAction $action)
     {
-        $response = $action->execute()->getEventTypeCollection();
+        $response = $action->execute(
```

Comment:
Please use proper naming, it already contains a collection. Or better, leave only the response here, and use `$response->getEventTypeCollection()` for the presenter.